.travis.yml

```
@@ -29,7 +29,7 @@ jobs:
     arch: arm64
```

```suggestion
    arch: arm64-graviton2
    vm: virt
```
Travis arm64 uses LXD by default. I guess that it causes the core number detected is quite large. In my experience, switching to use VM solves this problem. Please have a try.